### ISSUE TITLE
Support argparse ArgumentParser inheriting from parent parser

### ISSUE THREAD

**Issue by christabella** [guildai/guildai#147]
Hi! I have a parent parser and model-specific parser(s) as [recommended by PyTorch Lightning](<https://pytorch-lightning.readthedocs.io/en/latest/hyperparameters.html#multiple-lightning-modules>) and was wondering why only the model-specific arguments get recognized.
In `main.py`:
```parent_parser = ArgumentParser()
parent_parser.add_argument('--gpus', type=int, default=2)
parser = LatentModelPL.add_model_specific_args(parent_parser)
    hyperparams = parser.parse_args()
    print(hyperparams)
```

And defined in `LatentModelPL`:
```def add_model_specific_args(parent_parser):
    parser = ArgumentParser(parents=[parent_parser])
    parser.add_argument('--epochs', type=int, default=10)
    return parser
```

When I `print(hyperparams)` , both flags `epochs` and `gpus` show up.

However, when I run `guild run neural_process:train --help-op`, only the flags `epochs` is shown.
<sub>[View in Slack](https://guildai.slack.com/archives/C9PS9QS4A/p1584203012032600)</sub>

**Comment by christabella** [comment]
Maybe this part of the code is responsible for patching argparse and cloning the flags, and is not able to clone the parent flags for some reason https://github.com/guildai/guildai/blob/c60cd71ce10f7766dcfcc70037a980b1cdda8b35/guild/plugins/import_argparse_flags_main.py#L103

**Comment by gar1t** [comment]
This issue is resolved in master - the fix will be available in the next release.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report!
